### PR TITLE
Added voi explorer links

### PR DESCRIPTION
**Pull request by AryanPandeyy**
Issue #47

### CODE REVIEW

File: app/app-sandbox/src/pages/TokenAddress/index.js [hunk 1]
```diff
@@ -55,7 +55,12 @@ const Token = ({ address, token, transactions }) => {
           <code style={{ display: "inline-block" }}>
             {token.appId && (
               <span>
-                Id: <Link to={`/token/${token.appId}`}>{token.appId}</Link>
+                Id:{" "}
+                <a
+                  href={`https://voi.observer/explorer/application/${token.appId}/transactions/`}
```
Comment (temptemp3) [inline app/app-sandbox/src/pages/TokenAddress/index.js:60]: Let's have this link be return by a function that takes node, appId. You may leave the link for 'algorand' and 'algorand-testnet' not yet implemented.

Also, use target _blank and rel nofollow,noreferrer.

File: app/app-sandbox/src/pages/TokenAddress/index.js [hunk 2]
```diff
@@ -83,7 +88,21 @@ const Token = ({ address, token, transactions }) => {
...
+          For:{" "}
+          {NFDService.getNFDByAddress(address)?.[address]?.name ? (
+            <a
+              href={`https://voi.observer/explorer/account/${NFDService.getNFDByAddress(address)?.[address]?.name
+                }/transactions`}
+            >
+              {NFDService.getNFDByAddress(address)?.[address]?.name}
+            </a>
+          ) : (
+            <a
```
Comment (temptemp3) [inline app/app-sandbox/src/pages/TokenAddress/index.js:100]: Same comment as above here

**Comment by temptemp3** [comment]
Should also fix token id link on this page
https://github.com/temptemp3/arc-200/blob/cd8d99c3eac4e8f5687729efa091ac9f960a9c0f/app/app-sandbox/src/pages/Token/index.js#L353

**Comment by AryanPandeyy** [comment]
I'll work on the changes and will notify you once they are ready for review.

File: app/app-sandbox/src/pages/TokenAddress/index.js (hunk 1 above)
Comment (AryanPandeyy) [reply app/app-sandbox/src/pages/TokenAddress/index.js:60]: So there will be two functions. One for the appId and one for address?

**Comment by temptemp3** [comment]
Yes

**Pull request closed without merging**